msal/tests/Test.MSAL.NET.Unit.net45/CoreTests/InstanceTests/B2cAuthorityTests.cs

```diff
@@ -171,6 +163,8 @@ public void B2CMicrosoftOnlineCreateAuthority()
         [TestCategory("B2CAuthorityTests")]
         public void CanonicalAuthorityInitTest()
         {
+            var serviceBundle = ServiceBundle.CreateForProduction();
```

Comment:
>CreateForProduction [](start = 46, length = 19)

would really be great to understand why some tests does one thing and others the other thing.